.github/workflows/ci.yml

```diff
@@ -288,127 +288,50 @@ jobs:
       contents: 'read'
       id-token: 'write'
     steps:
-      - name: 'Checkout'
-        uses: 'actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11' # ratchet:actions/checkout@v4
-
-      - id: 'auth'
-        name: 'Authenticate to Google Cloud'
-        uses: 'google-github-actions/auth@35b0e87d162680511bf346c299f71c9c5c379033' # ratchet:google-github-actions/auth@v1
-        with:
-          workload_identity_provider: '${{ vars.WIF_PROVIDER }}'
-          service_account: '${{ vars.WIF_SERVICE_ACCOUNT }}'
-
-      - name: 'Setup gcloud'
-        uses: 'google-github-actions/setup-gcloud@e30db14379863a8c79331b04a9969f4c1e225e0b' # ratchet:google-github-actions/setup-gcloud@v1
+      - name: 'Deploy Artifact Pipeline'
+        uses: 'abcxyz/pkg/.github/actions/deploy-dataflow-pipeline@main'
```

Comment:
Can we add `# ratchet:exclude` to the end of all these actions?

Reply:
done.